src/eventmaster/plugins.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"crypto/subtle"
+	"encoding/hex"
+	"encoding/json"
+	"net/http"
+	"os"
+	"time"
+)
+
+var Plugins = map[string]Plugin{
+	"github": &GitHubPlugin{},
+}
+
+type Plugin interface {
+	ParseRequest(*http.Request, []byte) (map[string]interface{}, error)
+	ParseInfo(*http.Request, map[string]interface{}) (dc string, host string, user string, targetHostSet []string, tags []string, parentEventId string, eventTime int64, data map[string]interface{})
```

Comment:
The return of this should probably just be (*unsentevent, []topics) (or something similar)